CleanSwift/CleanSwift/Extension/UIViewController+Alert.swift

```diff
@@ -0,0 +1,21 @@
+// UIViewController+Alert.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import UIKit
+
+private enum Constants {
+    static let okTextString = "OK"
+}
+
+/// Вызов алерта
+extension UIViewController {
+    typealias Closure = (() -> ())?
```

Comment:
```suggestion
    typealias Closure = (() -> ())?
```
Возможно куда-нибудь в отдельный файл?